.eslintrc

```diff
@@ -75,5 +76,18 @@
         "node": true,
         "browser": true
     },
+    "overrides": [{
+        "files": ["katex.js", "src/**/*.js"],
+        "rules": {
+            "import/no-unresolved": 2,
+            "import/extensions": [2, "never"]
+        },
+        "env": {
+            "node": false
```

Comment:
This will warn if we try to use `process` inside of `src/**/*.js`?

Reply:
@kevinbarabash Yes.